contribs/gnodev/cmd/gnodev/txs.go

```diff
@@ -19,5 +21,37 @@ func parseTxs(txFile string) ([]std.Tx, error) {
 	}
 	defer file.Close()
 
-	return std.ParseTxs(context.Background(), file)
```

Comment:
You can create a similar helper in the gnoland package.

Reply:
Updated:

[7b9d87b](https://github.com/gnolang/gno/pull/2941/commits/7b9d87bcaf33355892cf8585a29a50e7ec89119b)